src/components/Dashboard/StaffNotation.vue

```diff
@@ -86,83 +86,78 @@
             <v-list-item
               data-type="court-order"
               @click="showCourtOrderDialog()"
-              :disabled="disabled || isAdminFreeze"
+              :disabled="!isAllowed(AllowableActions.COURT_ORDER)"
             >
               <v-list-item-title>
                 <span class="app-blue">Add Court Order</span>
               </v-list-item-title>
             </v-list-item>
 
             <v-list-item
+              v-if="isFirm"
               data-type="record-conversion"
               @click="goToConversionFiling()"
-              :disabled="disabled || !isFirm || isAdminFreeze"
+              :disabled="!isAllowed(AllowableActions.RECORD_CONVERSION)"
```

Comment:
My thought process here is that this is never applicable unless it's a firm, so only show it for a firm (line 97) and then enable/disable it accordingly (line 100).